docs/workflows/publish-host.rst

```diff
@@ -10,18 +10,18 @@ Create a Publisher
 Publishers contain extra settings for how to publish. You can use a Python publisher on any
 repository that contains Python content::
 
-$ http POST $BASE_ADDR/pulp/api/v3/publishers/python/ name=bar
+$ http POST $BASE_ADDR/pulp/api/v3/publishers/python/python/ name=bar
```

Comment:
hmmm. not pretty. lets change the publisher name to "warehouse" or "pypi"?

This is a non-blocking comment. Lets just file an issue and keep the PR as-is.